src/bento/dashboardTabData.js

```diff
@@ -1597,7 +1597,7 @@ export const tabContainers = [
       {
         dataField: 'toronto_childhood_cancer_staging',
         header: 'Toronto Childhood Cancer Staging',
-        display: false,
+        display: true,
```

Comment:
Where is it said that we need this to be visible on this ticket? Was this change intentional?

Reply:
This was merged , we had a lot of conflicts which I resolved a while back, and this change is from the conflict merge

Reply:
resolved